### DISCUSSION TITLE
如何配置 useSWR 的 SWRConfig ？

### DISCUSSION THREAD

**Discussion by jaybowang** [umijs/umi#10655] [Q&A] [answered]
SWR可以使用[SWRConfig全局配置](https://swr.vercel.app/docs/global-configuration)一些属性，类似[Redux的Provider](https://react-redux.js.org/api/provider)。
找了一下v4文档，没有看见如何全局配置，求助。

![image](https://user-images.githubusercontent.com/1948533/222342374-252dedf3-0d79-4af3-af95-2e100bef3381.png)

**Comment by fz6m** [comment, marked as answer]
全局 context 在 `src/app.tsx` [rootContainer](https://umijs.org/docs/api/runtime-config#rootcontainerlastrootcontainer-args) 配置。
